src/main/java/com/uwetrottmann/tmdb2/services/DiscoverService.java

```diff
@@ -28,8 +30,48 @@
+     * @param primary_release_date_lte <em>Optional.</em> Filter and only include movies that have a primary release date that is less than or equal to the specified value.
+     * @param release_date_gte         <em>Optional.</em> Filter and only include movies that have a release date (looking at all release dates) that is greater or equal to the specified value.
+     * @param release_date_lte         <em>Optional.</em> Filter and only include movies that have a release date (looking at all release dates) that is less than or equal to the specified value.
+     * @param vote_count_gte           <em>Optional.</em> Filter and only include movies that have a vote count that is greater or equal to the specified value.
+     * @param vote_count_lte           <em>Optional.</em> Filter and only include movies that have a vote count that is less than or equal to the specified value.
+     * @param vote_average_gte         <em>Optional.</em> Filter and only include movies that have a rating that is greater or equal to the specified value.
+     * @param vote_average_lte         <em>Optional.</em> Filter and only include movies that have a rating that is less than or equal to the specified value.
+     * @param with_cast                <em>Optional.</em> Only include movies that have one of the ID's added as an actor.
+     * @param with_crew                <em>Optional.</em> Only include movies that have one of the ID's added as a crew member.
+     * @param with_companies           <em>Optional.</em> Only include movies that have one of the ID's added as a production company.
+     * @param with_genres              <em>Optional.</em> Only include movies that have one of the ID's added as a genre.
+     * @param with_keywords            <em>Optional.</em> Only include movies that have one of the ID's added as a keyword.
+     * @param with_people              <em>Optional.</em> Only include movies that have one of the ID's added as a either a actor or a crew member.
+     * @param year                     <em>Optional.</em> A filter to limit the results to a specific year (looking at all release dates).
+     * @param without_genres           <em>Optional.</em> Genre ids that you want to exclude from the results.
+     * @param with_runtime_gte         <em>Optional.</em> Filter and only include movies that have a runtime that is greater or equal to a value.
+     * @param with_runtime_lte         <em>Optional.</em> Filter and only include movies that have a runtime that is less than or equal to a value.
+     * @param with_release_type        <em>Optional.</em> These release types map to the same values found on the movie release date method.
+     * @param with_original_language   <em>Optional.</em> ISO 639-1 string to filter results by their original language value.
+     * @param without_keywords         <em>Optional.</em> Exclude items with certain keywords.
```

Comment:
There have always been subtle documentation changes which do not necessarily change the API, like fixing typos or clarification. It is fine for now as the link to the docs is still included.